Channels/Insider/release_notes.md

```diff
@@ -13,19 +13,27 @@ Documentation Link: https://accessibilityinsights.io/docs/en/windows/overview
 - [Bug Fixes](#bug-fixes)
 
 #### New Features
-- Allow Single Sign On (SSO) in ADO issue filing [#1312](https://github.com/microsoft/accessibility-insights-windows/pull/1312), [#1316](https://github.com/microsoft/accessibility-insights-windows/pull/1316)
+- Display known potential framework issues separately to help you focus on problems that are directly in your control
+- Provide information about known framework issues, including potential workarounds or fixes
 
 #### Rule Updates
-- Name-based rules are no longer applied to `Text` elements that are children of a WPF `CheckBox` element [#648](https://github.com/microsoft/axe-windows/issues/648)
-- Improve the messaging of the `SplitButtonInvokeAndTogglePatterns` rule to more accurately reflect what the rule checks [#653](https://github.com/microsoft/axe-windows/issues/653)
-- Fix false positive triggered by `ControlShouldSupportExpandCollapsePattern` on a Win32 `SplitButton` control [#659](https://github.com/microsoft/axe-windows/issues/659)
+- Remove the `BoundingRectangleOnUWPMenuBar` rule ([Axe.Windows #698](https://github.com/microsoft/axe-windows/pull/698))
```

Comment:
Is it possible to add a little context for these first three removed rules? Specifically, were they removed entirely or replaced with something different?

I like how the "Edge Legacy" rules are grouped at the bottom and something similar here might help.

Reply:
We've reworded this after conversation with @nang4ally